specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-06-02-preview/examples/Fleets_PatchTags.json

```diff
@@ -0,0 +1,49 @@
+{
+  "parameters": {
+    "api-version": "2022-06-02-preview",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "fleetName": "fleet-1",
+    "If-Match": "dfjkwelr7384",
+    "parameters": {
+      "tags": {
+        "tier": "secure",
+        "env": "prod"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "ETag": "23ujdflewrj3="
+      },
+      "body": {
+        "id": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.ContainerService/fleets/fleet-1",
+        "name": "fleet-1",
+        "type": "Microsoft.ContainerService/fleets",
+        "systemData": {
+          "createdBy": "someUser",
+          "createdByType": "User",
+          "createdAt": "2022-03-23T05:40:40.657Z",
+          "lastModifiedBy": "someOtherUser",
+          "lastModifiedByType": "User",
+          "lastModifiedAt": "2022-03-23T05:40:40.657Z"
+        },
+        "tags": {
+          "tier": "secure",
+          "env": "prod"
+        },
+        "location": "East US",
+        "etag": "23ujdflewrj3=",
+        "properties": {
+          "hubProfile": {
+            "dnsPrefix": "dnsprefix1",
+            "fqdn": "dnsprefix1-abcd1234.flt.eastus.azmk8s.io",
+            "kubernetesVersion": "1.22.4"
+          },
+          "provisioningState": "Creating"
```

Comment:
patching tags should be done in sync without touching provisioningState?

Reply:
correct and it is. fixing the example. I used to have patch as an async operation. (will likely become async when we support other updates)

Reply:
fixed